perma_web/frontend/components/CreateLinkBatch.vue

```diff
@@ -88,10 +97,12 @@ const handleBatchCaptureRequest = async () => {
             })
 
         if (!response?.ok) {
-            throw new Error(response.statusText) // This is a placeholder for now
+            const errorResponse = await response.json()
```

Comment:
I think this will also want to call the new `getErrorResponse` utility, so this code doesn't break if the error is a non-JSON 502, etc. Am I reading that right?

Reply:
Yup! Was temporarily waiting on #3531 before pulling this in. 

Made some small code consistency changes in [12da131](https://github.com/harvard-lil/perma/pull/3537/commits/12da13129c61b8f3cb127318ddd7f555daf73c7f) and implemented the updated error handling for details fetches in [b1a4c79](https://github.com/harvard-lil/perma/pull/3537/commits/b1a4c797c95a1f7c17cda0f9ef679e12e0cfe31a)